Source/WebCore/platform/gamepad/cocoa/GameControllerGamepad.h

```diff
@@ -50,18 +48,12 @@ class GameControllerGamepad : public PlatformGamepad {
     const char* source() const final { return "GameController"_s; }
 
 private:
-    void setupAsExtendedGamepad();
-    void setupAsGamepad();
+    void setupElements();
```

Comment:
Elements seems bit generic, is this the best name for these?

Reply:
It's a term of art for "individual components on a controller", and shadows the GameController API.